meta/main.yml

```diff
@@ -15,4 +15,4 @@ galaxy_info:
   - clustering
 dependencies:
   - { role: smola.java, java_packages: 'oracle-java8-installer' }
-  - { role: galaxyproject.galaxy-os, galaxy_user_uid: 1001, docker_storage_location: /mnt/docker }
+  - { role: galaxyproject.galaxy-os, galaxy_user_uid: 10010, galaxy_user_gid: 10010, docker_storage_location: /mnt/docker }
```

Comment:
This is going to cause issues with FTP authentication - the uids needs to match with the ProFTPd conf: https://github.com/galaxyproject/ansible-cloudman-image/blob/master/templates/proftpd_default.j2#L78

Reply:
ok, what should we do about this? Now the latest NeCTAR image's ubuntu GID is 1001, so there's a conflict again. Shall I update proftpd too?

Reply:
The proftpd template needs to be parameterized anyway, so I guess that should be ok.

Reply:
I think so. Probably best to parameterize it with the same variable name?

I've also recently noticed that Nate has developed [a role for installing proftpd](https://github.com/galaxyproject/ansible-proftpd), although I don't think it's really worth it to incorporate it at this point the lifecycle of our overall playbook structure.

Reply:
I agree. Fixed in: https://github.com/galaxyproject/ansible-cloudman-image/pull/26/commits/42ee57a30eda2eda1a41a4b532918d39e033c371